src/app/tour/[id]/page.module.css

```
@@ -0,0 +1,50 @@
+.container {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  gap: 1rem;
+
+  .recommended {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    height: auto;
+    min-height: 90rem;
+    margin-block-end: 1rem;
+    margin-block-start: 5rem;
+
+    .recommended-header {
+      text-align: center;
+    }
+  }
+
+  .main {
+    .main-header {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      width: 100%;
+      padding: 1rem 5rem;
```

اینجا اصل Alignment تو UI رعایت نشده
بهتره 5rem رو ندید 

```suggestion
      padding-block: 1rem;
```